### ISSUE TITLE
Pop up Window with Error Messages

### ISSUE THREAD

**Issue by ccordoba12** [spyder-ide/spyder-notebook#107]
@JDepari commented on [Mon Aug 07 2017](https://github.com/spyder-ide/spyder/issues/4910)

## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

    module = _import_module_from_path(module_name, plugin_path)
    module = spec.loader.load_module(module_name)
  File "<frozen importlib._bootstrap_external>", line 399, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 823, in load_module
  File "<frozen importlib._bootstrap_external>", line 682, in load_module
  File "<frozen importlib._bootstrap>", line 248, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 608, in _exec
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
    from spyder_notebook.notebookplugin import NotebookPlugin as PLUGIN_CLASS
    from .utils.nbopen import nbopen, NBServerError
    from notebook import notebookapp
ImportError: cannot import name 'notebookapp'

## Version and main components

* Spyder Version: 3.2.0 
* Python Version: 3.6.1
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6

**Comment by ccordoba12** [comment]
@JDepari, how did you install `spyder-notebook`?

**Comment by JDepari** [comment]
I tried both using conda (conda install -c conda-forge spyder-notebook) and pip

**Comment by ccordoba12** [comment]
Is the notebook starting correctly for you when you run

    jupyter notebook

in a terminal?

**Comment by JDepari** [comment]
It is weird, it did not run anymore, before I installed spyder-notebook, jupyter notebook was running smoothly. Does it anything to do with the spyder-notebook installation ? Thanks

**Comment by ccordoba12** [comment]
We simply call `jupyter notebook` from Python to serve notebooks in Spyder.

**Comment by JDepari** [comment]
Alright, so probably  I need to make sure first that jupyter notebook works from terminal. Thanks.

**Comment by jonnojohnson** [comment]
I'm getting an error when I start Spyder that jupyter notebook server failed to start (or it's taking too long to start). But once spyder is launched I can open a notebook ok if I ignore the error. This was a fresh Anaconda install today on Win7. Jupyter notebook starts ok manually.

  File "C:\Users\marnj\AppData\Local\Continuum\Anaconda2\lib\site-packages\spyder\otherplugins.py", line 79, in _import_plugin
    module = _import_module_from_path(module_name, plugin_path)
  File "C:\Users\marnj\AppData\Local\Continuum\Anaconda2\lib\site-packages\spyder\otherplugins.py", line 101, in _import_module_from_path
    info = imp.find_module(module_name, [plugin_path])
ImportError: No module named spyder_notebook-0.1.1-py2.7.egg-info


## Version and main components

* Spyder Version: 3.2.1 
* Python Version: 2.7.13
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.5.0  :  1.5.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.20.1 (OK)
numpy >=1.7       :  1.12.1 (OK)
sphinx >=0.6.6    :  1.5.6 (OK)
rope >=0.9.4      :  0.9.4 (OK)
jedi >=0.9.0      :  0.10.2 (OK)
nbconvert >=4.0   :  5.1.1 (OK)
sympy >=0.7.3     :  1.0 (OK)
cython >=0.21     :  0.25.2 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.3.0 (OK)
pylint >=0.25     :  1.6.4 (OK)

```

**Comment by dalthviz** [comment]
Hi @jonnojohnson please see the issue #112 basically you need to delete the file folder `spyder_notebook-0.1.1-py2.7.egg-info`. In your case it should be at  `C:\Users\marnj\AppData\Local\Continuum\Anaconda2\lib\site-packages\`.

**Comment by jonnojohnson** [comment]
Thanks, that did solve it partly but I'm still getting the dialog box that tells me the notebook server didn't start or is too slow. Then when Spyder opens up I get a message in the IPython console that the kernel died and is restarting. When IPython finally restarts I get 2 In [1]: lines ie:

In [1]:

In [1]:

**Comment by jonnojohnson** [comment]
Also when I then open a new IPython console, each time the kernel dies and restarts with the same In [1]: behavior.

**Comment by dalthviz** [comment]
The dialog box and the behavior you are expiriencing with the IPython console are related normally with a lack of resources (RAM or CPU) when opening Spyder. What are your specs @jonnojohnson? Also, I think that your issue is different from the one here so please open a new issue :)

**Comment by JDepari** [comment]
Hi, 
These were steps I did to solve the issues:
1. Uninstall Anaconda
2. Install again but now I change the folder installation to drive other drive other than C (in my case, I installed it on D drive). When I installed Anaconda, I did not use Administrator user and I also set that the the installation type was set to only my account. 
3. Install spyder-notebook using conda command.
With these steps, I now can run Spyder as well as spyder notebook smoothly, no more pop up window appears.

Version and components:
- Windows 10 Education
- Anaconda 4.4.0
- Python 3.6 64 bits
- Spyder 3.2.1
- Sypder notebook v0.1.1
